crud/src/main/resources/json-schema/findRequest.json

```diff
@@ -45,11 +45,15 @@
          },
          "from": {
              "type": "integer",
-             "description": "Alternateive to 'range' allowing 'from' and 'to' to be set independently."
+             "description": "Alternative to 'range' allowing 'from' and 'to' to be set independently."
          },
          "to": {
              "type": "integer",
-             "description": "Alternateive to 'range' allowing 'from' and 'to' to be set independently."
+             "description": "Alternative to 'range' allowing 'from' and 'to' to be set independently."
+         },
+         "maxResults": {
+             "type": "integer",
+             "description": "Alternative to 'range' allowing 'from' and 'to' to be set independently."
```

Comment:
Description doesn't tell me that if I set "to" and "maxResults" the value of "to" is used, not "maxResults".  Could we update it to be clearer about when maxResults can be used?